packages/grafana-icons/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@grafana/icons",
+  "version": "0.0.0",
+  "description": "Icons for Grafana",
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-icons"
+  },
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "main": "dist/index.js",
+    "module": "dist/esm/index.js",
+    "types": "dist/index.d.ts",
+    "access": "public"
+  },
+  "files": [
+    "./dist",
+    "./README.md",
+    "./CHANGELOG.md",
+    "./LICENSE_APACHE2"
+  ],
+  "scripts": {
+    "clean": "rimraf ./dist ./compiled ./package.tgz ./src/icons-gen",
+    "generate": "yarn clean && npx @svgr/cli ./svg && mv ./src/icons-gen/index.ts ./src",
+    "typecheck": "tsc --emitDeclarationOnly false --noEmit",
+    "lint": "eslint --ext .ts,.tsx ./src",
+    "prettier:check": "prettier --check --list-different=false --log-level=warn \"**/*.{ts,tsx,scss,md,mdx,json}\"",
+    "build": "tsc -p ./tsconfig.build.json && rollup -c rollup.config.ts",
+    "bundle": "rollup -c rollup.config.ts"
+  },
+  "devDependencies": {
+    "@grafana/tsconfig": "^1.3.0-rc1",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/plugin-typescript": "^11.1.6",
+    "@svgr/babel-plugin-remove-jsx-attribute": "^8.0.0",
+    "@svgr/cli": "^8.1.0",
+    "@svgr/plugin-jsx": "^8.1.0",
+    "@svgr/plugin-prettier": "^8.1.0",
+    "@svgr/plugin-svgo": "^8.1.0",
+    "@types/node": "^20.11.10",
+    "@types/react": "^18.2.48",
+    "@types/react-dom": "^18",
+    "esbuild": "^0.20.0",
+    "prettier": "^3.2.4",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "rimraf": "^5.0.5",
+    "rollup": "2.79.1",
+    "rollup-plugin-dts": "^6.1.0",
+    "rollup-plugin-esbuild": "5.0.0",
+    "rollup-plugin-node-externals": "5.0.0",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.3.3"
```

Comment:
Can you please check that that version of these dependencies match the root workspace package.json? I think especially the esbuild and typescript versions should match the same.